src/main/scala/com/databricks/spark/sql/perf/results.scala

```diff
@@ -113,9 +113,12 @@ case class MLParams(
     numTestExamples: Option[Long] = None,
     numPartitions: Option[Int] = None,
     // *** Specialized and sorted by name ***
+    elasticNetParam: Option[Double] = None,
+    family: Option[String] = None,
+    link: Option[String] = None,
+    k: Option[Int] = None,
```

Comment:
@yhuai What are restrictions on changing the order of arguments for case classes used within BenchmarkResult?  Will it break reading from existing log files using the previous schema?

Maybe it's not a problem if the last commit's version was not yet deployed though.